src/app/technical-record/dimensions/dimensions.component.spec.ts

```diff
@@ -24,24 +26,32 @@ describe('DimensionsComponent', () => {
     fixture = TestBed.createComponent(DimensionsComponent);
     component = fixture.componentInstance;
     const tecRec: TechRecord = {
-                                frontAxleTo5thWheelMin: 456,
-                                frontAxleTo5thWheelMax: 660,
-                                frontAxleTo5thWheelCouplingMin: 100,
-                                frontAxleTo5thWheelCouplingMax: 150,
-                                dimensions: {
-                                            length: 100,
-                                            width: 100,
-                                            axleSpacing: [
-                                              { axles: '1', value: 100},
-                                              { axles: '2', value: 100},
-                                              { axles: '3', value: 100},
-                                            ]
-                                            }
-                              } as TechRecord;
+        frontAxleToRearAxle: 450,
```

Comment:
Extract into reusable like this with this `const getTechRecord = (): TechRecord => {
  return {
    frontAxleTo5thWheelMin: 456,
    frontAxleTo5thWheelMax: 660,
    frontAxleTo5thWheelCouplingMin: 100,
    frontAxleTo5thWheelCouplingMax: 150,
    dimensions: TESTING_UTILS.mockDimensions({
      axleSpacing: [
        { axles: '1', value: 100 },
        { axles: '2', value: 100 },
        { axles: '3', value: 100 }
      ]
    })
  } as TechRecord;
};`

Reply:
done